subsys/bluetooth/host/hci_core.c

```diff
@@ -4276,7 +4276,10 @@ int bt_le_set_chan_map(uint8_t chan_map[5])
 	struct bt_hci_cp_le_set_host_chan_classif *cp;
 	struct net_buf *buf;
 
-	if (!IS_ENABLED(CONFIG_BT_CENTRAL)) {
+	if (!(IS_ENABLED(CONFIG_BT_CENTRAL)
+	    | IS_ENABLED(CONFIG_BT_ISO_BROADCASTER)
+	    | IS_ENABLED(CONFIG_BT_ISO_CENTRAL)
+	    | IS_ENABLED(CONFIG_BT_PER_ADV))) {
```

Comment:
With the "Channel Classification Enhancement" added in 5.3 it may be useful for the ACL peripheral to set the channel map as well, we should maybe add that one while we are at it?

Reply:
Are these really supposed to be bitwise or and not logical or?

Reply:
It should definitely be `||` instead of `|` :)

Reply:
> Are these really supposed to be bitwise or and not logical or?

Oops, my mistake!